client/modules/App/AppReducer.js

```diff
@@ -1,27 +1,27 @@
-// Import Actions
-import { TOGGLE_ADD_POST } from './AppActions';
+import { TOGGLE_ADD_POST, TOGGLE_EDIT_POST } from "./AppActions";
 
-// Initial State
 const initialState = {
   showAddPost: false,
+  showEditPost: false
```

Comment:
add comma on the end ;) when you expand this object another time, git will show you only line you add, not both ;)